.setup/data/courses/sample.yml

```diff
@@ -336,7 +474,80 @@ gradeables:
   gradeable_config: python_simple_homework
   g_bucket: homework
 
+- g_id: grading_only_peer_team_homework
+  eg_peer_grading: true
+  components:
+  - gc_lower_clamp: 0
+    gc_is_peer: true
+    gc_default: 2
+    gc_max_value: 2
+    gc_upper_clamp: 2
+    gc_title: Read Me
+    marks:
+      - gcm_note: Full Credit
+        gcm_points: 0
+      - gcm_note: Minor errors in Read Me
+        gcm_points: -1
+      - gcm_note: Major errors in Read Me or Read Me missing
+        gcm_points: -2
+  - gc_lower_clamp: 0
+    gc_default: 5
+    gc_is_peer: true
+    gc_max_value: 5
+    gc_upper_clamp: 5
+    gc_title: Coding Style
+    marks:
+      - gcm_note: Full Credit
+        gcm_points: 0
+      - gcm_note: Code is unreadable
+        gcm_points: -5
+      - gcm_note: Code is very difficult to understand
+        gcm_points: -3
+      - gcm_note: Code is difficult to understand
+        gcm_points: -1
+  - gc_lower_clamp: 0
+    gc_default: 5
+    gc_is_peer: true
+    gc_max_value: 5
+    gc_upper_clamp: 5
+    gc_title: Documentation
+    marks:
+      - gcm_note: Full Credit
+        gcm_points: 0
+      - gcm_note: No documentation
+        gcm_points: -5
+      - gcm_note: Very little documentation or documentation makes no sense
+        gcm_points: -3
+      - gcm_note: Way too much documentation and/or documentation makes no sense
+        gcm_points: -1
+  - gc_lower_clamp: 0
+    gc_default: 0
+    gc_is_peer: true
+    gc_max_value: 0
+    gc_upper_clamp: 5
+    gc_title: Extra Credit
+    marks: 
+      - gcm_note: No Credit
+        gcm_points: 0
+      - gcm_note: Extra credit done poorly
+        gcm_points: 2
+      - gcm_note: Extra credit is acceptable
+        gcm_points: 5
+  eg_submission_due_date: 1972-01-01 23:59:59
+  eg_submission_open_date: 1971-01-01 23:59:59
+  g_grade_due_date: 9998-12-31 23:59:59
+  g_grade_released_date: 9998-12-31 23:59:59
+  g_grade_start_date: 1973-01-01 23:59:59
+  eg_regrade_request_date: 9999-01-06 23:59:59
+  eg_team_assignment: true
+  eg_max_team_size: 3
+  g_ta_view_start_date: 1970-01-01 23:59:59
+  g_title: Closed Peer Team Homework
+  gradeable_config: python_simple_homework
+  g_bucket: homework
+
 - g_id: closed_team_homework
```

Comment:
This should be renamed if it is going to be a peer gradeable (closed_team_peer_homework or something similar)

Reply:
Added a peer team gradeable and a non peer team gradeable with same configuration as "Closed Team Homework"

Reply:
done!